src/NetaceaConfig.json

```diff
@@ -1,7 +1,9 @@
 {
   "apiKey": "",
   "secretKey": "",
-  "kinesisStreamName": "",
-  "KinesisAccessKey": "",
-  "secretsManagerSecretId": ""
+  "kinesis": {
+    "kinesisStreamName": "",
+    "kinesisAccessKey": "",
+    "kinesisSecretKey": ""
+  }
```

Comment:
Do they still need prefixing with `kinesis` if they're already in an object named `kinesis`?

Reply:
Unfortunately yes, was the easiest way within the types I could make it backward compatible in the module!